sdk/cognitiveservices/Knowledge.QnAMaker/tests/QnAMakerRuntimeTests.cs

```diff
@@ -19,7 +19,12 @@ public void QnAMakerRuntimeGenerateAnswerTest()
                 HttpMockServer.Initialize(this.GetType(), "QnAMakerRuntimeGenerateAnswerTest");
 
                 var client = GetQnAMakerRuntimeClient(HttpMockServer.CreateInstance());
-                var answer = client.Runtime.GenerateAnswerAsync("8758c6af-fa29-4e03-a517-9c36927f558f", new QueryDTO(question: "good afternoon")).Result;
+                var queryDTO = new QueryDTO();
+                queryDTO.StrictFiltersCompoundOperationType = StrictFiltersCompoundOperationType.OR;
+                queryDTO.StrictFilters = new List<MetadataDTO>();
+                queryDTO.StrictFilters.Add(new MetadataDTO("question", "good afternoon"));
+                queryDTO.StrictFilters.Add(new MetadataDTO("question", "good morning"));
+                var answer = client.Runtime.GenerateAnswerAsync("8758c6af-fa29-4e03-a517-9c36927f558f", queryDTO).Result;
```

Comment:
whr is data file for test?

Reply:
QnAMakerRuntimeTests\QnAMakerRuntimeGenerateAnswerTest.json